tests/typ/compute/foundations.typ

```diff
@@ -87,7 +87,7 @@ image("/tiger.jpg")
 #show raw: it => eval("[" + it.text + "]")
 
 ```
-#show emph: _ => image("/giraffe.jpg")
```

Comment:
Shouldn't this be allowed?

Reply:
Yes you're right, I thought this would get fixed once patterns are allowed in parameters, but it won't.